src/main/java/frc/robot/subsystems/drive/PhoenixDrive.java

```diff
@@ -182,17 +202,37 @@ public void setGoalSpeeds(ChassisSpeeds goalSpeeds, boolean fieldCentric) {
                                 < 1e-10
                         && Math.abs(goalSpeeds.omegaRadiansPerSecond) < 1e-10;
         Logger.recordOutput("Drive/idling", idling);
-        if (idling) {
+
+        // sticks zeroed out will cause false idle when wanting alignment
+        if (idling && !aligning) {
```

Comment:
Could we try
```java
if (idling && !(aligning && !isDriveAligned))
```
or something, so that we don't go into X brake when aligning but done moving?
Or will a FieldCentricFacingAngle request not go into X brake?

Reply:
@aidnem i can't find anything in docs, I say we just try out what's here and if it goes into x brake tonight I can update it real quick and then get merged after testing

Reply:
@linglejack06 did we ever end up testing this

Reply:
@aidnem no we didnt, i sepnt rest of time trying to fix shooter as one of the motors wasnt working (an electrical issue i think). Justin stayed past nine and I believe shooting works well now. So we can now intake and shoot.

Reply:
@linglejack06 Good to know, thanks!